public/javascripts/exportOption.js

```diff
@@ -0,0 +1,122 @@
+function exportOption(initialType) {
+    var queryString = '';
+    var type = initialType;
+
+    function pdfOption() {
+        if (type !== 0) {
+            return m('div', [
+                m('.radio', [
+                    m('label', [
+                        m('input', {
+                            'type': 'radio',
+                            'name': 'optradio2',
+                            onclick: function() {
+                                queryString = '?type=1'
+                            },
+                            config: function(elem, isInit) {
+                                if (!isInit) {
+                                    if (type === 1) {
+                                        $(elem).prop('checked', 'checked')
+                                        queryString = '?type=1'
+                                    }
+                                }
+                            }
+                        })
+                    ], '1 language per slide')
+                ]),
+                m('.radio', [
+                    m('label', [
+                        m('input', {
+                            'type': 'radio',
+                            'name': 'optradio2',
+                            onclick: function() {
+                                queryString = '?type=2'
+                            },
+                            config: function(elem, isInit) {
+                                if (type === 2) {
+                                    if (!isInit) {
+                                        $(elem).prop('checked', 'checked')
+                                        queryString = '?type=2'
+                                    }
+                                }
+                            }
+                        })
+                    ], '2 languages per slide')
+                ]),
+                m('.radio', [
+                    m('label', [
+                        m('input', {
```

Comment:
refactor each input element so it becomes a function (name, queryString, type, label) {} instead of listing it out.
I would also consider using a different word than type because it is overloaded in programmer speech. it would be better to call it languagesPerSlide.